choose.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ *
+ */
+class Choose
+{
+  /***
+  レア度を抽選する
+  ***/
+
+  public static function choose_rare($rarity){
```

Comment:
PHP7からタイプヒンティングという機能が追加されました。
IDEと組み合わせることによる恩恵がすごく大きいので使って行きたい。
それから関数の引数リストと中括弧の間にはスペースあるいは改行を入れるのが一般的です。

```suggestion
  public static function choose_rare(array $rarity)
  {
```

PHPのコーディングガイドがあるので参考にしてみると良いと思います。
http://www.infiniteloop.co.jp/docs/psr/psr-2-coding-style-guide.html

まぁ実際にはプロジェクトで採用されているコーディングスタイルに準拠することになると思います。